.changeset/great-eyes-tan.md

```diff
@@ -0,0 +1,54 @@
+---
+"@guardian/cdk": major
+---
+
+Access logging for Application Load Balancers (ALBs) is now enabled by default.
+
+The Application Load Balancer (ALB) access logs are helpful during incident triage.
+Therefore, they are now enabled by default.
+
+Previously users of the `GuEc2App`, `GuNodeApp`, `GuPlayApp `, `GuPlayWorkerApp` and `GuEc2AppExperimental` patterns could opt-in to this logging via the `accessLogging` property and configure the S3 prefix.
+
+This property is now removed and replaced with a new optional boolean property `withAccessLogging` which defaults to `true`.
+- When `true` the ALB will have access logs enabled, configured to write to the account's S3 bucket
+using a specific prefix for compatibility with the Athena database created in https://github.com/guardian/aws-account-setup.
```

Comment:
Maybe this isn't the right place to mention it, but I think it'd be useful to write the name of the database somewhere.

```suggestion
using a specific prefix for compatibility with the `gucdk_access_logs` database created in Athena via https://github.com/guardian/aws-account-setup.
```

Reply:
Updated. I've also added this to the docstrings too.